apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx

```diff
@@ -128,6 +131,7 @@ class LibraryCreationDialog extends React.Component {
       libraryJson,
       error => {
         console.warn(`Error publishing library: ${error}`);
```

Comment:
Still want this console warn now that there's UI for it?

Reply:
Jessie and I chatted and we decided yes, because then we can help debug if someone writes in with issues.